specification/schemaregistry/SchemaRegistry/client.tsp

```diff
@@ -0,0 +1,28 @@
+import "@azure-tools/typespec-client-generator-core";
+import "./main.tsp";
+ 
+using Azure.ClientGenerator.Core;
+
+@@access(Azure.SchemaRegistry.Version, Access.internal);
+@@projectedName(Azure.SchemaRegistry.Version, "client", "SchemaVersion");
+@@access(Azure.SchemaRegistry.SchemaGroup, Access.internal);
+
+// csharp modifications
```

Comment:
The ContentTypeEnum was added for C# and the operations were made internal for C# as well. I made them internal for everyone b/c I don't think any of the languages expose the operations anyway, b/c we have wrappers. The models are private b/c they were being generated under the public models namespace otherwise, which we didn't want. I made them private for everyone, b/c I don't think these models were being publicly exposed for them either. I can update them to public if anyone asks! Thoughts?